Corgibytes.Freshli.Cli/CommandRunners/CacheCommandRunner.cs

```diff
@@ -33,5 +34,37 @@ public override int Run(CachePrepareCommandOptions options)
             return success ? 0 : 1;
         }
     }
-}
 
+    public class CacheDestroyCommandRunner : CommandRunner<CacheDestroyCommandOptions>
+    {
+        public CacheDestroyCommandRunner(IServiceProvider serviceProvider, Runner runner)
+            : base(serviceProvider, runner)
+        {
+
+        }
+
+        public override int Run(CacheDestroyCommandOptions options)
+        {
+            // Skip prompt if the --force flag is passed
+            if (options.Force)
+                goto Destroy;
```

Comment:
Please use `{` and `}` explicitly. And rewrite this logic so that it does not rely on `goto`.

Reply:
Thanks for making the change.